Mathlib/GroupTheory/SpecificGroups/Cyclic.lean

```diff
@@ -137,7 +137,7 @@ theorem Subgroup.eq_bot_or_eq_top_of_prime_card {G : Type*} [Group G] {_ : Finty
     (H : Subgroup G) [hp : Fact (Fintype.card G).Prime] : H = ⊥ ∨ H = ⊤ := by
   classical
   have := card_subgroup_dvd_card H
-  rwa [Nat.dvd_prime hp.1, ← Nat.card_eq_fintype_card, ← Nat.card_eq_fintype_card,
+  rwa [Nat.card_eq_fintype_card (α := G), Nat.dvd_prime hp.1, ← Nat.card_eq_fintype_card,
```

Comment:
Not sure why this is necessary? Looks like you're just exchanging the order of two rewrites.

Reply:
It's necessary because `this` is in terms of `Nat.card` and `hp` is in terms of `Fintype.card`, so we have to switch `this` over to `Fintype.card` before apply `Nat.dvd_prime`.

Reply:
(This will all get cleaned up once the file is switched over to `Nat.card`).

Reply:
I see, I missed that you removed an arrow.

maintainer merge